source/nordic_sdk/components/ble/peer_manager/id_manager.h

```diff
@@ -229,6 +229,31 @@ ret_code_t im_wlist_create(pm_peer_id_t        * p_peer_ids,
  */
 bool im_address_resolve(ble_gap_addr_t const * p_addr, ble_gap_irk_t const * p_irk);
 
+/**@brief Function for calculating the ah() hash function described in Bluetooth core specification

```

Comment:
Can you make a note here to say this is special modification needed in mbed. and write down the reason you need it. Make it prominent as well, enclose it in something like  ==========. Next time when we make the port to a newer version (v11 is on the horizon) the porter will know to keep this modification in.